### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by mirii1994**
- v1.5
    - Update client version(v1.8).
    - `sub_account`:
        - **Breaking changes:**
            - Deprecated attribute `utilization_settings`. Use `frequency_minutes` and `utilization_enabled` instead. 
        - Added attributes `flexible` and `reserved_daily_gb`.
        - Refactor tests.
        - Refactor code to use Terraform's retry.
    - `endpoint`:
        - **Breaking changes:**
            - Types naming.
        - New endpoints for OpsGenie, ServiceNow and Microsoft Teams.
        - Fix bug for `body_template` for `custom` type ([#70](https://github.com/logzio/terraform-provider-logzio/issues/70)).
        - Refactor tests.
        - Refactor code to use Terraform's retry.
    - `alerts_v2`:
        - Fix bug for `filter_must`,`filter_must_not` ([#82](https://github.com/logzio/terraform-provider-logzio/issues/82))
        - Refactor tests.
        - Refactor code to use Terraform's retry.
    - `drop_filter`:
        - Improve tests.

### CODE REVIEW

File: logzio/common.go [hunk 1]
```diff
@@ -1,6 +1,10 @@
 package logzio
```
Comment (Doron-Bargo) [inline logzio/common.go:1]: can you move it to be under utils?

File: logzio/parsers.go [hunk 2]
```diff
@@ -3,6 +3,7 @@ package logzio
 import (
```
Comment (Doron-Bargo) [inline logzio/parsers.go:3]: can you move it to be under utils?

File: logzio/validators.go [hunk 3]
```diff
@@ -4,6 +4,8 @@ import (
 	"fmt"
```
Comment (Doron-Bargo) [inline logzio/validators.go:4]: can you move it to be under utils?

**Review by Doron-Bargo — CHANGES_REQUESTED** [review]
Amazing work 
Some cosmetics changes

File: docs/data-sources/endpoint.md [hunk 4]
```diff
@@ -39,3 +39,7 @@ Logz.io integrates with:
 * [Datadog](https://docs.logz.io/api/#operation/updateDataDog)
 * [VictorOps](https://docs.logz.io/api/#operation/updateVictorops)
 * [Custom integration](https://docs.logz.io/api/#operation/updateCustom)
+* [OpsGenie](https://docs.logz.io/api/#operation/createOpsGenie).
```
[the remark this answers is not in the document]
Comment (mirii1994) [reply docs/data-sources/endpoint.md:42]: Probably happened when my eyes were really tired. Should be create. Fixing.

File: logzio/resource_subaccount.go [hunk 5]
```diff
@@ -230,26 +216,86 @@ func resourceSubAccountDelete(d *schema.ResourceData, m interface{}) error {
...
+		AccountName:            d.Get(subAccountName).(string),
+		Flexible:               strconv.FormatBool(d.Get(subAccountFlexible).(bool)),
+		ReservedDailyGB:        float32(d.Get(subAccountReservedDailyGb).(float64)),
+		MaxDailyGB:             float32(d.Get(subAccountMaxDailyGB).(float64)),
+		RetentionDays:          int32(d.Get(subAccountRetentionDays).(int)),
+		Searchable:             strconv.FormatBool(d.Get(subAccountSearchable).(bool)),
+		Accessible:             strconv.FormatBool(d.Get(subAccountAccessible).(bool)),
+		SharingObjectsAccounts: sharingObjectAccounts,
+		DocSizeSetting:         strconv.FormatBool(d.Get(subAccountDocSizeSetting).(bool)),
+		UtilizationSettings: sub_accounts.AccountUtilizationSettingsCreateOrUpdate{
```
[the remark this answers is not in the document]
Comment (mirii1994) [reply logzio/resource_subaccount.go:283]: In the API, `utilizationSettings` is an object that contains two fields: `frequencyMinutes`, `utilizationEnabled`.
I wanted to flatten and simplify the configuration for the user, so I deprecated `utilization_settings`, and instead the user just needs to use `frequency_minutes` and `utilization_enabled` (they already used those fields, but inside `utilization_settings`). Behind the scenes, in the code, I insert those values to UtilizationSettings.

**Review by Doron-Bargo — APPROVED** [review]
Amazing

**Comment by caphrim007** [comment]
i'm eagerly awaiting this release. awesome job @mirii1994

**Comment by akvamalin** [comment]
Hey guys,
thanks for implementing missing connectors (in particular MS Teams Webhook). Is there any plan when to merge this PR?
Looking forward for the merge,
Best regards.

**Pull request merged**